be/src/exec/pipeline/operator.h

```diff
@@ -59,10 +60,8 @@ class Operator {
 
     RuntimeProfile* get_runtime_profile() const { return _runtime_profile.get(); }
 
-    std::string get_name() const {
-        std::stringstream ss;
-        ss << _name + "_" << this;
-        return ss.str();
+    virtual std::string get_name() const {
+        return strings::Substitute("$0_$1($2)", _name, this, is_finished() ? "X" : "O");
```

Comment:
The X or O I think aren't clear, I think could directly use `finished` or `unfinished`.